es-app/src/CollectionSystemManager.cpp

```diff
@@ -67,13 +70,15 @@ CollectionSystemManager::~CollectionSystemManager()
 	removeCollectionsFromDisplayedSystems();
 
 	// iterate the map
-	for(std::map<std::string, CollectionSystemData>::iterator it = mCustomCollectionSystemsData.begin() ; it != mCustomCollectionSystemsData.end() ; it++ )
+	for (auto & collection : mCollectionSystems)
+
+	//for(std::map<std::string, CollectionSystem>::iterator it = mCustomCollectionSystemsData.begin() ; it != mCustomCollectionSystemsData.end() ; it++ )
```

Comment:
I am a fan of using the 'new' `for ( auto thing : setOfThings){ ... }` syntax, and will be changing to that where-ever I can.

Reply:
Looks good to me.